test/kcl/action_handler_tests.js

```diff
@@ -79,7 +79,7 @@ describe('action_handler_tests', function() {
 
   it('should write action to stdout', function(done) {
     actionHandler.sendAction({action : 'initialize', shardId : 'shardId-000001'}, function(err) {
-      should.equal(err, undefined);
+      should.equal(err, null);
```

Comment:
Why is this change being made

Reply:
Tests running locally and on workflow return null instead of undefined. Not sure how long the test has been broken. Maybe a change in a node version changed something